doc/nrf/ug_nrf9160.rst

```diff
@@ -110,9 +110,14 @@ See the `Mobile network operator certifications`_ for more information.
 There are two ways to update the modem firmware:
 
 Full upgrade
-  A full upgrade of the modem firmware requires a wired connection.
-  The upgrade is done through the nRF Connect Programmer, which is part of `nRF Connect for Desktop`_.
+  A full upgrade of the modem firmware can be done through either a wired  or a wireless connection.
+  When using a wired connection, the upgrade is done through the nRF Connect Programmer, which is part of `nRF Connect for Desktop`_.
   See `Updating the nRF9160 DK cellular modem`_ in the nRF Connect Programmer User Guide for instructions.
+  It is also possible to use the Simple Management Protocol (SMP) to provide an interface over UART which enables the device to do full modem firmware updates.
```

Comment:
This is a bit misleading, since nRF Connect for Desktop leverages this protocol as well. The sample provides a python script which use the nrfjprog API directly. This is what the customers would use to see how the update procedure is implemented on the PC side. I'm not sure how to phrase it, but in broad lines:
- Wired update is done using SMP
- nRF Connect for Desktop can be used for performing full modem update regardless of existing firmware
- There is a sample fmfu_smp_svr_sample which shows how to integrade this full modem update over SMP support in an application.
- This sample also provides a python script which shows how to implement the PC side of the update.

The difference between nrf Connect for Desktop and the python script in the sample is that nrf connect for desktop will work regardless of the existing firmware on the device, while the script requires the specific sample to be programmed.

Reply:
Please check the rephrased section